modules/boot/src/utils/hook.cpp

```diff
@@ -197,14 +200,27 @@ int saveInjectHook(void* i_scene) {
 // Hook to disable save inject flags after dScnPly phase_4 is run
 int endSaveInjectHook(void* i_scene) {
     int rt = dScnPly__phase_4Trampoline(i_scene);
+    s16 scene = fpcM_GetName(i_scene);
 
+    if (scene == PROC_OPENING_SCENE ) {
+        g_PreLoopListener->addListener(GZ_endlessNightOnTitle);
+    }
     if (SaveManager::s_injectSave || SaveManager::s_injectMemfile) {
         SaveManager::s_injectSave = false;
         SaveManager::s_injectMemfile = false;
     }
     
     return rt;
 }
+int dScnPly_DeleteHook(void* i_scene) {
+    s16 scene = fpcM_GetName(i_scene);
+
+    if (scene == PROC_OPENING_SCENE) {
+        g_PreLoopListener->removeListener(GZ_endlessNightOnTitle);
+    }
```

Comment:
same with this, should probably be hooked somewhere else that's more relevant to the title screen and not just general gameplay state